apps/web/src/client/components/ui/TextInput.tsx

```diff
@@ -13,12 +13,12 @@ export function TextInput({ field, value, placeholder, maxLength, onBlur, onChan
   maxLength?: number,
   onBlur?: () => void,
   onChange: (x: string) => void,
-  isSecret?: boolean
+  isSecret?: boolean,
+  autoComplete?: string,
+  isApiKey?: boolean
```

Comment:
I feel like instead of having `isSecret` and `isApiKey`, it would be better to have a `type` parameter of type `"text" | "password" | "secret"`, where `"text"` would be the default. `secret` would work as if `isSecret = true; isApiKey = true`, `password` would work like `isSecret = true; isApiKey = false`, and `text` would work like `isSecret = false; isApiKey = false`.